src/main/java/net/earthcomputer/clientcommands/command/GammaCommand.java

```diff
@@ -0,0 +1,36 @@
+package net.earthcomputer.clientcommands.command;
+
+import static com.mojang.brigadier.arguments.DoubleArgumentType.getDouble;
+import static com.mojang.brigadier.arguments.DoubleArgumentType.doubleArg;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.addClientSideCommand;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.server.command.CommandManager.argument;
+import static net.minecraft.server.command.CommandManager.literal;
+
+import com.mojang.brigadier.CommandDispatcher;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+
+public class GammaCommand {
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("cgamma");
+
+        dispatcher.register(literal("cgamma")
+            .then(argument("gamma", doubleArg())
+                .executes(ctx -> setGamma(ctx.getSource(), getDouble(ctx, "gamma")))));
+    }
+
+    private static int setGamma(ServerCommandSource source, double gamma) {
+        MinecraftClient.getInstance().options.gamma = gamma;
+
+        Text feedback = new TranslatableText("commands.cgamma.success", gamma);
+        sendFeedback(feedback);
+
+        return 1;
```

Comment:
```suggestion
        return 0;
```
Unless you got something more useful to return.

Reply:
Would basing the return value off of whether the gamma was changed work?

Reply:
Sure

Reply:
If the gamma started at 0.3, and the user ran `/cgamma 0.5` twice, the command would return `1` then `0`.